src/geometry/WorkingGeometry.ts

```diff
@@ -93,4 +110,25 @@ export class WorkingGeometry {
             colors: bakedColors
         };
     }
+
+    /**
+     * Update the current vertices, faces, and control points performing the computation of merging
```

Comment:
The current wording is somewhat confusing - are the control points performing the computation? Please reword to something like
```
Updates the current vertices, faces, and points to merge the child objects.
```